store/conf/attrs/zimbra-attrs.xml

```diff
@@ -1822,7 +1822,7 @@ TODO - add support for multi-line values in globalConfigValue and defaultCOSValu
 </attr>
 
 <attr id="335" name="zimbraFeatureSharingEnabled" type="boolean" cardinality="single" optionalIn="account,cos" flags="accountInfo,accountInherited">
-  <defaultCOSValue>TRUE</defaultCOSValue>
+  <defaultCOSValue>FALSE</defaultCOSValue>
```

Comment:
we should check how this will behave in case of upgrade scenario
if customer is upgrading from 9.x.x to 10.1.x then existing default cos will have value TRUE only not FALSE, which is different then our expectation

Reply:
If they have license entitlement they can enbled it back. Lets discuss this tomorrow

Reply:
enabling for large user base will be problem